content/en/docs/about/community.md

```diff
@@ -56,18 +56,24 @@ The following calendars are maintained by [Kubeflow Working Groups](#kubeflow-wo
 | Calendar | Link |
 | --- | --- |
 | Kubeflow Community | [Google Calendar](https://calendar.google.com/calendar/embed?src=kubeflow.org_7l5vnbn8suj2se10sen81d9428%40group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/kubeflow.org_7l5vnbn8suj2se10sen81d9428%40group.calendar.google.com/public/basic.ics), [Git Repo](https://github.com/kubeflow/community/tree/master/calendar))
-| Kubeflow Release Team | [Google Calendar](https://calendar.google.com/calendar/embed?src=c_c5i4tlc61oq2kehbhv9h3gveuo%40group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/c_c5i4tlc61oq2kehbhv9h3gveuo%40group.calendar.google.com/public/basic.ics))
 | Kubeflow AutoML WG | [Google Calendar](https://calendar.google.com/calendar/embed?src=d49nsiefsnfgkf91o0u8h2jhk4%40group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/d49nsiefsnfgkf91o0u8h2jhk4%40group.calendar.google.com/public/basic.ics))
-| Kubeflow Manifests WG | [Google Calendar](https://calendar.google.com/calendar/embed?src=c_mhv1kfv9na0pdile4qju6o5v60%40group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/c_mhv1kfv9na0pdile4qju6o5v60%40group.calendar.google.com/public/basic.ics))
-| Kubeflow Notebooks WG | [Google Calendar](https://calendar.google.com/calendar/embed?src=c_ptktc8ulh50db8bqvr9fd4ubqs%40group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/c_ptktc8ulh50db8bqvr9fd4ubqs%40group.calendar.google.com/public/basic.ics))
```

Comment:
Just to confirm: Should people use Kubeflow community calendar for Notebooks and Manifests meeting?

Reply:
These calendars no longer exist, all meetings are in the main "Kubeflow Community Calander" now, which is managed by: https://github.com/kubeflow/community/tree/master/calendar

With the one exception of the AutoML WG, which I have left in this PR (but we should move that meeting to the main community calendar in the future, I have put a topic in the next Community Call about that).

Reply:
Sounds good. We can discuss more in the next community meeting, and create another PR based on that discussion result.